server/index.js

```diff
@@ -47,6 +47,98 @@ app.get('/api/products/:productId', (req, res, next) => {
     });
 });
 
+app.get('/api/cart', (req, res, next) => {
+  const select = `
+          select "c"."cartItemId",
+          "c"."price",
+          "p"."productId",
+          "p"."image",
+          "p"."name",
+          "p"."shortDescription"
+        from "cartItems" as "c"
+       join "products" as "p" using ("productId")
+       where "c"."cartId" = $1
+  `;
+  db.query(select, [req.session.cardId])
+    .then(result => res.json(result.rows))
+    .catch(err => {
+      console.error(err);
+      next(err);
+    });
+});
```

Comment:
This is missing a conditional, please check the instructions for this code very carefully. It's describe in two places in the steps.